network-store-server/src/main/resources/schema.sql

```diff
@@ -413,6 +413,23 @@ create index on threeWindingsTransformer (networkUuid, variantNum, voltageLevelI
 create index on threeWindingsTransformer (networkUuid, variantNum, voltageLevelId3);
 
 
+CREATE TABLE IF NOT EXISTS temporaryLimit (
+    equipmentId VARCHAR(255) NOT NULL,
+    equipmentType VARCHAR(255),
+    networkUuid VARCHAR(255) NOT NULL,
```

Comment:
`networkUuid` is a `uuid` type elsewhere

Reply:
Using a UUID instead of a String brings some issues, I'd rather keep it that way for the moment.